src/pcapi/core/bookings/api.py

```diff
@@ -178,3 +178,12 @@ def compute_confirmation_date(
         )
         return latest_date_between_earliest_date_in_cancellation_period_and_booking_creation
     return None
+
+
+def update_confirmation_dates(
+    bookings: typing.List[Booking], beginning_datetime: datetime.datetime
```

Comment:
```suggestion
    bookings: List[Booking], beginning_datetime: datetime.datetime
```
Et mettre l'import en haut.